src/renderer.js

```diff
@@ -0,0 +1,32 @@
+const { isInteger } = require("lodash");
+
+const render = require("./render");
+const toPromise = require("./consumers/promise");
+const toNodeStream = require("./consumers/node-stream");
+
+
+class Renderer {
+  constructor (jsx, sequence) {
```

Comment:
nit: it's not really JSX being passed here as its just a syntax extension and we're passing the result of the JSX call. It might be better to call it `element` or something.